src/react/components/common/assetPreview/tfrecordAsset.tsx

```diff
@@ -54,6 +54,12 @@ export class TFRecordAsset extends React.Component<IAssetProps, ITFRecordState>
         }
     }
 
+    private onError = (e: React.SyntheticEvent<Element>) => {
+        if (this.props.onError && this.state.tfRecordImage64 !== "") {
```

Comment:
Is there a way to check that `e` itself is empty? I thought you mentioned that the error had no message

Reply:
So I actually don't think that was the issue, but rather that we should ignore the error initially when the asset preview tries to load an empty string.  The error is correct still I think and we should catch if the tfrecord image can't load, just not because of an empty string.